README.md

```diff
@@ -6,7 +6,7 @@ and its open source variant, CyanWorlds.com Engine. For more information, visit
 
 Dependencies
 ------------
-- [Blender](http://blender3d.org) - 3d modelling problem
+- [Blender](http://blender3d.org) - 3D modeling software
```

Comment:
In _proper_ English, "modelling" is spelt with 2 Ls :trollface:

Reply:
It's true.  It's one of those words that frustrates me with being partially incorrect in both forms, according to different rules... ;)

Do we have a standard preference for which English to use?  Despite my inclination, I assume en-US is consistent with most of the codebase.

Reply:
We prefer Southeastern US English in Korman. :wink: